include/deal.II/grid/grid_tools.h

```diff
@@ -1164,7 +1169,9 @@ namespace GridTools
     const GridTools::Cache<dim, spacedim> &                cache,
     const std::vector<Point<spacedim>> &                   local_points,
     const std::vector<std::vector<BoundingBox<spacedim>>> &global_bboxes,
-    const double                                           tolerance = 1e-10);
+    const double                                           tolerance = 1e-10,
+    const std::vector<bool> &                              marked_vertices = {},
+    const bool enforce_unique_mapping = true);
```

Comment:
I guess the different ordering of the parameters in comparison to the one of the internal function is a bit annoying but we cannot change that.

Reply:
I was thinking about the same thing. I think even if we could change it without problems within ```deal.II``` (what I am not sure about), it might break user code. Thus, I refrained from this.